docker_compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: shopadmin
+      POSTGRES_PASSWORD: testpassword
+      POSTGRES_DB: shop
+    ports:
```

Comment:
Since your `app` is literally dependent on the DB, you'd need to add a `healthcheck`.